qdrant-landing/content/articles/discovery-search.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Discovery needs context" #required
+short_description: Discover points by constraining the space
+description: Qdrant released a new functionality that lets you constrain the space in which a search is performed, relying only pure vectors #required
+social_preview_image: /articles_data/discovery-search/social_preview.jpg # This image will be used in social media previews, should be 1200x630px. Required.
+small_preview_image: /articles_data/discovery-search/icon.svg # This image will be used in the list of articles at the footer, should be 40x40px
+preview_dir: /articles_data/discovery-search/preview # This directory contains images that will be used in the article preview. They can be generated from one image. Read more below. Required.
+weight: -100 # This is the order of the article in the list of articles at the footer. The lower the number, the higher the article will be in the list.
+author: Luis Cossío # Author of the article. Required.
+author_link: https://coszio.github.io # Link to the author's page. Required.
+date: 2023-12-11T13:00:00+03:00 # Date of the article. Required.
```

Comment:
I presume you'll want to change the date

Reply:
Yes, as soon as I know the publication date 😄